### PR TITLE
Preferences menu now supports Halloween races

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

Halloween will break if I don't finish this so there, it's finished :)

## Why It's Good For The Game

I think we all want Halloween to work, right? Okay, just making sure

## Changelog

:cl:
fix: Preferences menu now supports Halloween races
/:cl:

### CODE REVIEW

File: tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/vampire.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      name: "The Thirst",
+      description: "In place of eating, vampires suffer from The Thirst. \
+      Thirst of what? Blood! Their tongue allows them to grab people and drink \
+      their blood, and they will die if they run out. As a note, it doesn't \
+      matter whose blood you drink, it will all be converted into your blood \
+      type when consumed.",
+    }],
+  },
+  lore: [
+    "\"I'm not doing the bat trick. It's self deprecating. Okay, fine, but I'm not doing it again!\" - Count Baz, before venting into security for the third time this week.",
```
Comment (RaveRadbury) [inline tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/vampire.ts:39]: Do we want to encourage this behavior by offering it to the players as a starting prompt like this? This is the only lore that does such a thing.

File: tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/shadow.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+const Shadowperson: Species = {
+  description: "Victims of a long extinct space alien. Their flesh is a sickly \
+    seethrough filament, their tangled insides in clear view. Their form \
+    is a mockery of life, leaving them mostly unable to work with others under \
+    normal circumstances.",
+  features: {
+    good: [{
+      icon: "moon",
+      name: "Shadowborn",
+      description: "Their skin blooms in the darkness. All kinds of damage- \
```
Comment (Mothblocks) [inline tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/shadow.ts:12]: Should probably be commas.

File: tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/shadow.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      no matter how extreme- will heal over time as long as there is no light.",
+    }, {
+      icon: "eye",
+      name: "Nightvision",
+      description: "Their eyes have adapted to the night. Their red eyes can \
+      see in the dark with no problems.",
```
Comment (Mothblocks) [inline tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/shadow.ts:18]: Very odd wording here, I'd replace with just "Their eyes, adapted to the night, can see in the dark with no problems."

File: tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/skeleton.ts [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import { Species } from "./base";
+
+const Skeleton: Species = {
+  description: "A rattling skeleton! They descend upon Space Station Thirteen \
```
Comment (Mothblocks) [inline tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/skeleton.ts:4]: We call it 13 in the nuke ops description, but doesn't matter so much.

File: tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/skeleton.ts [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    neutral: [],
+    bad: [{
+      icon: "user-times",
+      name: "Limbs Easily Dismembered",
+      description: "Skeletons limbs are not secured well, and as such they are \
+        easily dismembered.",
+    }],
+  },
+  lore: [
+    "Skeletons want to be feared again! Their presence in media has been destroyed, or at least that's what they firmly believe. They're always the first thing faught in an RPG, they're flanderized into pun rolling JOKES, and it's really starting to get to them. You could say they're deeply RATTLED. Hah.",
```
Comment (Mothblocks) [inline tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/skeleton.ts:29]:
```suggestion
    "Skeletons want to be feared again! Their presence in media has been destroyed, or at least that's what they firmly believe. They're always the first thing fought in an RPG, they're Flanderized into pun rolling JOKES, and it's really starting to get to them. You could say they're deeply RATTLED. Hah.",
```

File: tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/vampire.ts [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+        not needing to breathe or eat, but do not get many of the \
+        environmental immunities involved with being fully undead.",
+    }, {
+      icon: "recycle",
+      name: "Bat Form",
+      description: "Vampires can become bats. Bats are very weak, but \
+        are great for escaping bad situations. They can also travel through \
+        vents, giving Vampires a lot of access. Just remember that access \
+        doesn't equal permission, and people may be unhappy with you showing \
+        up anywhere!",
```
Comment (Mothblocks) [inline tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/vampire.ts:25]:
```suggestion
        up uninvited!",
```

I just think it's kinda funny considering that thing about vampires not being allowed in if they're not invited.

File: tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/zombie.ts [hunk 7]
```diff
@@ -0,0 +1,45 @@
+import { Species } from "./base";
+
+const Zombie: Species = {
+  description: "A rotting zombie! They descend upon Space Station Thirteen \
+    Every year to spook the crew! \"Sincerely, the Zombies!\"",
+  features: {
+    good: [{
+      icon: "user-plus",
+      name: "Limbs Easily Reattached",
+      description: "Zombies limbs are easily readded, and as such do not \
```
Comment (Mothblocks) [inline tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/zombie.ts:10]:
```suggestion
      description: "A zombie's limbs are easily readded, and as such do not \
```

File: tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/zombie.ts [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+      name: "No Body Temperature",
+      description: "Having long since departed, zombies do not have anything \
+        regulating their body temperature anymore. This simply means that \
+        their environment decides their temperature, which they don't mind at \
+        all until it gets a bit too hot.",
+    }],
+    bad: [{
+      icon: "user-times",
+      name: "Limbs Easily Dismembered",
+      description: "Zombies limbs are not secured well, and as such they are \
```
Comment (Mothblocks) [inline tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/zombie.ts:31]:
```suggestion
      description: "A zombie's limbs are not secured well, and as such they are \
```

**Review by Mothblocks — CHANGES_REQUESTED** [review]

File: tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/vampire.ts (hunk 1 above)
Comment (Mothblocks) [reply tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/vampire.ts:39]: to be fair, this lore text is a bit funny

**Comment by Mothblocks** [comment]
No problems in code, just some second opinions on the writing.

File: tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/skeleton.ts (hunk 4 above)
Comment (tralezab) [reply tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/skeleton.ts:4]: UNITY

File: tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/vampire.ts (hunk 1 above)
Comment (tralezab) [reply tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/vampire.ts:39]: I'll add some more actual lore so this isn't JUST the joke, that'll prolly do enough

**Review by Mothblocks — APPROVED** [review]
Clean functions declarations/calls formatting. Make the UI simple enough for basic technology users.

**Comment by tralezab** [comment]
I am very sorry i didn't have a team am only doing this alone
Or can you be explain more
Thanks you so much ChoiceCoin

**Comment by TiviPlus** [comment]
Thanks for this great opportunity

File: tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/vampire.ts [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+      icon: "recycle",
+      name: "Bat Form",
+      description: "Vampires can become bats. Bats are very weak, but \
+        are great for escaping bad situations. They can also travel through \
+        vents, giving Vampires a lot of access. Just remember that access \
+        doesn't equal permission, and people may be unhappy with you showing \
+        up uninvited!",
+    }],
+    neutral: [],
+    bad: [{
```
Comment (Fikou) [inline tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/vampire.ts:28]: doesnt mention them DYING the moment they enter the chapel

**Pull request merged**